runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/MergeSegmentsRequest.java

```diff
@@ -0,0 +1,38 @@
+package org.corfudb.protocols.wireprotocol.orchestrator;
+
+import static org.corfudb.protocols.wireprotocol.orchestrator.OrchestratorRequestType.MERGE_SEGMENTS;
+
+import java.nio.charset.StandardCharsets;
+
+import lombok.Getter;
+
+/**
+ * An orchestrator request to merge all segments in the cluster.
+ * Created by Zeeshan on 2019-02-06.
+ */
+public class MergeSegmentsRequest implements CreateRequest {
```

Comment:
It's kind of confusing MergeSegmentsRequest from MergeSegments (that implements WorkflowRequest) is one for redundancy and merge and the other only for merge? why two requests? I thought we only needed the workflow request, can we make this cleaner? or can you please explain this. thanks

Reply:
MergeSegments is the abstraction which actually sends the message to the orchestrator.
While the MergeSegmentsRequest is just the request message which gets sent.